generation/consolidate_config.sh

```diff
@@ -96,10 +96,10 @@ removeArtifactVersion 'dependency' 'google-api-services-cloudresourcemanager'
 removeArtifactVersion 'dependency' 'google-api-services-storage'
 removeArtifactVersion 'dependency' 'google-cloud-storage'
 removeArtifactVersion 'plugin' 'maven-checkstyle-plugin'
-removeArtifactVersion 'dependency' 'mockito-all'
+removeArtifactVersion 'dependency' 'mockito-core'
```

Comment:
I don't think any changes to this file are needed. This was relevant when we were migrating from split repos.

Reply:
Reverted this change!